grails-app/migrations/0.8.x/changelog-2023-09-21-1330-alter-table-requisition-add-rejectedby-field.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="kchelstowski" id="210920231330-1">
```

Comment:
Just a question, why are you numbering change sets from 1 instead of 0? 🤔

Reply:
I don’t know 😄

Reply:
Blasphemous behavior. :smiling_imp: